### DISCUSSION TITLE
Maintained source packages?

### DISCUSSION THREAD

**Discussion by pmetzger** [mikebrady/shairport-sync#1875] [Ideas]
If there were source packages for shairport-sync (and nqptp) for Debian, the AUR, and a few other things, it would not be necessary to manually build and reinstall when updates happen. Instead, one could just use the system tools to do it given the .dsc file or what have you with a command or two.

**Comment by mikebrady** [comment]
Thanks for the suggestion. If there was a way to automate this, say by using GitHub Actions, it would be great! If anyone has any hints or examples, they’d be welcome.

**Comment by pmetzger** [comment]
You could certainly test that the builds for the various platforms work using Github actions. That's the easy part. The harder part is getting source packages for several operating systems.
